git_pw/config.py

```diff
@@ -25,6 +26,9 @@ def _get_config(key):
     except subprocess.CalledProcessError:
         output = ''
 
+    if os.environ.get('PW_%s' % key.upper()) is not None:
```

Comment:
I was hoping to use [click's built-in support for this](http://click.pocoo.org/5/options/#values-from-environment-variables). This would allow us to add application-level `--username` and `--password` options as a more prominent way of configuring this. The `auto_envvar_prefix` option seems to be exactly what we're looking for. Could you either drop this particular commit, or rework to use the click options?